helm_deploy/values-prod.yaml

```diff
@@ -18,7 +18,6 @@ generic-service:
     SDS_EARLY_RELEASE_HINTS_FEATURE_TOGGLE: "true"
     SUPPORT_INACTIVE_SENTENCES_AND_ADJUSTMENTS: "false"
     TORERA_OFFENCE_TO_MANUAL_JOURNEY: "true"
-    HDC_365_FEATURE_TOGGLE: "false"
```

Comment:
@joelstobart-moj Is this what you wanted me to do?

Reply:
Yes - will deploy to live tomorrow morning